lib/system/generic/xlnx/CMakeLists.txt

```diff
@@ -8,4 +8,10 @@ collect (PROJECT_LIB_HEADERS sys.h)
 
 collect (PROJECT_LIB_SOURCES irq.c)
 
+include(CheckSymbolExists)
+check_symbol_exists(SDT "bspconfig.h" HAS_SDT)
+if (HAS_SDT)
```

Comment:
Could you provide a more explicite name for file and defines?
replace `sdt.h` by `sys_devicetree.h` ( or something similar)
replace `SDT by SYSTEM_DT`
replace `HAS_SDT by HAS_SYSTEM_DT`

Reply:
SDT is the symbol provided in the new BSP

the other 2 can do

Reply:
> Could you provide a more explicite name for file and defines? replace `sdt.h` by `sys_devicetree.h` ( or something similar) replace `SDT by SYSTEM_DT` replace `HAS_SDT by HAS_SYSTEM_DT`

I replaced sdt.h and HAS_SDT as suggested.
I kept SDT symbol as this is provided by a header we don't control.